lib/Imagine/Imagick/Effects.php

```diff
@@ -81,7 +81,7 @@ public function colorize(ColorInterface $color)
         }
 
         try {
-            $this->imagick->colorizeImage((string) $color, new \ImagickPixel(sprintf('rgba(%d, %d, %d, 1)', $color->getRed(), $color->getGreen(), $color->getBlue())));
+            $this->imagick->colorizeImage((string) $color, new \ImagickPixel(sprintf('rgba(%d, %d, %d, 1)', $color->getRed(), $color->getGreen(), $color->getBlue())), true);
```

Comment:
Should be false (see https://github.com/mkoppanen/imagick/blob/63e9f6bc5184d601298aaa54aebca0ed7cf0d12b/imagick_class.c#L8930-L8932)